cves/kernel/CVE-2019-20054.yml

```diff
@@ -224,71 +206,33 @@ subsystem:
       * can have digits, and _-@/
 
     Can be multiple subsystems involved, in which case you can make it an array
+
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
-interesting_commits:
-  question: |
-    Are there any interesting commits between your VCC(s) and fix(es)?
-
-    Use this to specify any commits you think are notable in some way, and
-    explain why in the note.
-
-    Example interesting commits:
-      * Mentioned as a problematic commit in the past
-        e.g. "This fixes regression in commit xys"
-      * A significant rewrite in the git history
-      * Other commits that fixed a similar issue as this vulnerability
-      * Anything else you find interesting.
-  commits:
-  - commit:
-    note:
-  - commit:
-    note:
-i18n:
-  question: |
-    Was the feature impacted by this vulnerability about internationalization
-    (i18n)?
-
-    An internationalization feature is one that enables people from all
-    over the world to use the system. This includes translations, locales,
-    typography, unicode, or various other features.
-
-    Answer should be true or false
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
-sandbox:
+discovered:
+  answer: "It seems that YueHaibing <yuehaibing@huawei.com> was running the 
+  Syzkaller to generate a report, and used it to find this bug. Syzkaller is 
+  an kernel fuzzer https://github.com/google/syzkaller"
```

Comment:
I like that you provided a link for the fuzzer in case people want more information